graph.js

```diff
@@ -181,12 +221,21 @@ app
     }
 
     let client = jayson.Client.http(requestUrl)
-    if (cmd.apiKey !== undefined) {
-      client.options.headers = { Authorization: 'Bearer ' + cmd.apiKey }
-    }
 
     let logger = new Logger(0, { verbosity: getVerbosity(app) })
 
+    if(cmd.accessToken !== undefined) {
+      client.options.headers = { Authorization: 'Bearer ' + cmd.accessToken }
+    } else {
+      keytar.getPassword('graphprotocol-auth', cmd.node).then((accessToken) => {
```

Comment:
`cmd.node` isn't going to work. All it takes is `http://some-ip:8020/` to be passed in instead of `http://some-ip:8020` and it won't work.

Let's do this:
```
let node = normalizeNodeUrl(cmd.node)
```
before we pass it in here, and also when we store the access token in the first place.
What normalizeNodeUrl should do is return the same URL regardless of whether there was a leading slash, leading or trailing whitespace etc. Ideally use https://nodejs.org/api/url.html for this.